python/lsst/afw/display/interface.py

```diff
@@ -523,27 +526,26 @@ def pan(self, colc=None, rowc=None, origin=afwImage.PARENT):
         """!Pan to (rowc, colc); see also zoom"""
 
         self.zoom(None, colc, rowc, origin)
-
+    
```

Comment:
Sorry I didn't know that this is the convention. My text editor, Textmate, adds whitespace by default that matches the current indentation level (for the class in this case). Personally I like this style *much* better but I'll be more careful in the future.